tests/bulk_create/tests.py

```diff
@@ -121,11 +121,15 @@ def test_batch_same_vals(self):
         self.assertEqual(Restaurant.objects.count(), 2)
 
     def test_large_batch(self):
-        with override_settings(DEBUG=True):
```

Comment:
So weird that we were overriding  `DEBUG=True` here without ever looking up `connection.queries` later on.

Reply:
Oops, I didn't mean to commit any test changes.